### PR TITLE
Makes announcement hearable even if deaf

### PR DESCRIPTION
**Pull request by breadhunt**
## About The Pull Request
Lets deaf, crit, or dead humans hear all announcements, like in CM.

## Why It's Good For The Game
Currently, TGMC is extremely momentum based. If you start losing, it can be hard to stop it. Part of this problem is how common deafness is. 
When someone is deaf, they cannot hear anything, sometimes for an extended period of time. You get absolutely NO information, except what is _visible_ on your screen, and that's absolutely crushing. ESPECIALLY when some things deafen you for _half a fucking hour at times_. Deafness is an **incredibly** strong debuff.
But this PR doesn't change that, and that's why I think it's balanced. The only thing this allows command to do, is _ensure_ that everybody heard something. Nothing else really fills that role currently, and from how organised CM is at times, I think it's something that'd make here a lot more cohesive and overall fun to play.

TL;DR:
- Raises marine awareness, and probably competence
- Lets command ensure everybody has heard something, rather than having to guess
- Gives better opportunities for shipside roles, e.g RO, to let people know what's going on
- Makes communicating with your team easier, in the team-based game

## Changelog
:cl:
balance: Announcements (including command ones) can now be heard while deaf, in crit, or dead.
/:cl:

### CODE REVIEW

**Comment by BraveMole** [comment]
Nice Why It's Good For The Game, take exemple folks

**Review by TiviPlus — APPROVED** [review]
